### ISSUE TITLE
"Discard all unstaged changes" warning not clear enough

### ISSUE THREAD

**Issue by stevage** [divmain/GitSavvy#309]
I just lost some untracked files, by using the `D` "discard all unstaged changes" option. I read the description of the option, and I read the warning:

>You are about to discard all unstaged changes.  This is a destructive action.  
>Would you like to proceed?  
>(you can disable this prompt in GitSavvy.sublime-settings)

That warning did not convey to me that Git was about to delete my *untracked* files as well. That is, a couple of files that had been in my working directory for quite a while, and had never been checked in.

Could I suggest:

>You are about to discard all unstaged changes, and delete all untracked files. This is a destructive action.
>Are you ABSOLUTELY SURE you want to do this?

I find it hard to imagine that anyone would ever want to disable this prompt - really, press shift-D and delete all your changes with no prompt? (And if they really do, they can figure out how...)

**Comment by divmain** [comment]
Actually, there was quite a bit of discussion back when the prompt was introduced.  So I'm going to leave in that bit about disabling it in your settings file.  However, clarifying that untracked files will be discarded seems a reasonable idea.  Will add!

**Comment by divmain** [comment]
I've made the change.  Let me know if you feel like this is "good enough", or if you have strong feelings to the contrary.

**Comment by stevage** [comment]
OIC, it's the same message structure for "little d" and "big D" discard.

I have no remaining strong feelings, other than gratitude for your work on an excellent tool. Thanks. :)

(Aside: I wrote [this epic rant](http://stevebennett.me/2012/02/24/10-things-i-hate-about-git/) four years ago. I've invested quite a lot of effort in trying to use Git, including through various interfaces. Git-Savvy actually makes Git pretty bearable for me - so for that, I'm extremely grateful.)
